.evergreen/run-kerberos-tests.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o errexit  # Exit the script with error if any of the commands fail
+
+export PROJECT_DIRECTORY="$(pwd)"
+NODE_ARTIFACTS_PATH="${PROJECT_DIRECTORY}/node-artifacts"
+export NVM_DIR="${NODE_ARTIFACTS_PATH}/nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+
+# set up keytab
+mkdir -p "$(pwd)/.evergreen"
+touch "$(pwd)/.evergreen/krb5.conf.empty"
+export KRB5_CONFIG="$(pwd)/.evergreen/krb5.conf.empty"
+echo "Writing keytab"
+# DON'T PRINT KEYTAB TO STDOUT
+set +o verbose
+DECODE='-d'
+OS=`uname`
+if [[ $OS == "Darwin" ]]; then
```

Comment:
This `if` statement is set up to switch on the value `DECODE`, why not just do the work inside and save some lines:
```
if [[ "$OSTYPE" == "darwin"* ]]; then
    echo $KEYTAB | base64 -D > "$(pwd)/.evergreen/drivers.keytab"
else
    echo $KEYTAB | base64 -d > "$(pwd)/.evergreen/drivers.keytab"
fi
```